etna/datasets/tsdataset.py

```diff
@@ -1122,3 +1126,76 @@ def info(self, segments: Optional[Sequence[str]] = None) -> None:
         # print the results
         result_string = "\n".join(lines)
         print(result_string)
+
+    def to_train_dataloader(self, encoder_length, decoder_length, columns_to_add, batch_size: int = 1):
+        from torch.utils.data import DataLoader
+
+        df = self.to_pandas(flatten=True)
+
+        ts_segments = [df_segment for _, df_segment in df.groupby("segment")]
+        ts_samples = [
+            i
+            for dict_segment in ts_segments
+            for i in make_samples(dict_segment, encoder_length, decoder_length, columns_to_add)
+            if not i["encoder_real"].isnan().any()
+        ]
+
+        return DataLoader(ts_samples, batch_size=batch_size)
+
+    def to_test_dataloader(self, encoder_length, decoder_length, columns_to_add, batch_size: int = 1):
```

Comment:
Also typings. They are needed not only for parameters but for output of the methods as well.